discussion/src/main/java/org/openedx/discussion/presentation/comments/DiscussionCommentsFragment.kt

```diff
@@ -673,3 +679,4 @@ private val mockComment = DiscussionComment(
     users = mapOf(),
     isAuthor = false,
 )
+
```

Comment:
Please remove this extra blank line as well. Ideally, there should be only one blank line at the end of the file.